doc/cephadm/adoption.rst

```diff
@@ -137,6 +137,9 @@ Adoption process
 
    where *<placement>* can be a simple daemon count, or a list of
    specific hosts (see :ref:`orchestrator-cli-placement-spec`).
+   Example::
+   
+     # ceph orch apply rgw myorg us-east-1 --placement="2 myhost1 myhost2"
 
```

Comment:
Can we also clarify what this means, whether it deploys 2 RGWs each to myhost1 & myhost2 or 2 RGWs in total, one each in both the hosts

Reply:
Hm. that's why there is a link to https://docs.ceph.com/docs/master/mgr/orchestrator/#placement-specification above. 

Do you think we need to explicitly that that here again?